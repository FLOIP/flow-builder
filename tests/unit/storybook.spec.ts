// tests/unit/storybook.spec.js
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: './config/storybook/',
});
