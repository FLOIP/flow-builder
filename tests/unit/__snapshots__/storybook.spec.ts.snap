// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ConsoleIo/Print Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots ConsoleIo/Print Block Existing Data Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots ConsoleIo/Print Block Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots ConsoleIo/Read Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots ConsoleIo/Read Block Existing Data Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots ConsoleIo/Read Block Existing Data Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Case Block Styled Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Case Block Styled Existing Data Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Log Block Plain Default 1`] = `<!---->`;

exports[`Storyshots Core/Log Block Plain Existing Data Block 1`] = `<!---->`;

exports[`Storyshots Core/Log Block Styled Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Log Block Styled Existing Data Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Log Block Styled Existing Data Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Output Block Styled Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Core/Run Another Flow Block Styled Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Flow/Flow Editor Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots Flow/Flow Editor Existing Data Pre Filled 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots InteractionDesigner/Toolbar Default 1`] = `
<div>
  <div
    class="viamo-app-container flow-builder-sidebar-editor-container"
    id="viamo-app"
  >
    <div
      class="container-full"
    >
      <div
        class="main-content-container"
      >
        <section
          id="tree-app"
        >
          <div
            class="edit-tree"
            id="tree-interaction-designer"
          >
            <div
              class="interaction-designer-contents panel panel-default"
            >
              <!---->
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots InteractionDesigner/Toolbar Edit Flow 1`] = `
<div>
  <div
    class="viamo-app-container flow-builder-sidebar-editor-container"
    id="viamo-app"
  >
    <div
      class="container-full"
    >
      <div
        class="main-content-container"
      >
        <section
          id="tree-app"
        >
          <div
            class="edit-tree"
            id="tree-interaction-designer"
          >
            <div
              class="interaction-designer-contents panel panel-default"
            >
              <!---->
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots InteractionDesigner/Toolbar With Extra Buttons Slot 1`] = `
<div>
  <div
    class="viamo-app-container flow-builder-sidebar-editor-container"
    id="viamo-app"
  >
    <div
      class="container-full"
    >
      <div
        class="main-content-container"
      >
        <section
          id="tree-app"
        >
          <div
            class="edit-tree"
            id="tree-interaction-designer"
          >
            <div
              class="interaction-designer-contents panel panel-default"
            >
              <!---->
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots InteractionDesigner/Toolbar With Grouped Buttons Slot 1`] = `
<div>
  <div
    class="viamo-app-container flow-builder-sidebar-editor-container"
    id="viamo-app"
  >
    <div
      class="container-full"
    >
      <div
        class="main-content-container"
      >
        <section
          id="tree-app"
        >
          <div
            class="edit-tree"
            id="tree-interaction-designer"
          >
            <div
              class="interaction-designer-contents panel panel-default"
            >
              <!---->
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots InteractionDesigner/Toolbar With Save Button 1`] = `
<div>
  <div
    class="viamo-app-container flow-builder-sidebar-editor-container"
    id="viamo-app"
  >
    <div
      class="container-full"
    >
      <div
        class="main-content-container"
      >
        <section
          id="tree-app"
        >
          <div
            class="edit-tree"
            id="tree-interaction-designer"
          >
            <div
              class="interaction-designer-contents panel panel-default"
            >
              <!---->
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots InteractionDesigner/Toolbar Without Resource Editor Toggle 1`] = `
<div>
  <div
    class="viamo-app-container flow-builder-sidebar-editor-container"
    id="viamo-app"
  >
    <div
      class="container-full"
    >
      <div
        class="main-content-container"
      >
        <section
          id="tree-app"
        >
          <div
            class="edit-tree"
            id="tree-interaction-designer"
          >
            <div
              class="interaction-designer-contents panel panel-default"
            >
              <!---->
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots MobilePrimitives/Message Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Message Block Existing Data Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Message Block Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Numeric Response Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Numeric Response Block Existing Data For All Modes 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Numeric Response Block Existing Data For Ivr Only 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Numeric Response Block Existing Data For Text Only 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Numeric Response Block Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Open Response Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Open Response Block Existing Data For All Modes 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Open Response Block Existing Data For Ivr Only 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Open Response Block Existing Data For Text Only 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/Open Response Block Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/SelectOneResponseBlock Existing Data 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/SelectOneResponseBlock In Flow Builder 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/SelectOneResponseBlock Ivr Only 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/SelectOneResponseBlock More Languages 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/selectManyResponseBlock Existing Data 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/selectManyResponseBlock In Flow Builder 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/selectManyResponseBlock Ivr Only 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots MobilePrimitives/selectManyResponseBlock More Languages 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots SmartDevices/Location Response Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots SmartDevices/Location Response Block Existing Data Pre Filled 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots SmartDevices/Location Response Block Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots SmartDevices/Photo Response Block Default 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots SmartDevices/Photo Response Block Existing Data Pre Filled 1`] = `
<div>
  <!---->
</div>
`;

exports[`Storyshots SmartDevices/Photo Response Block Non Starting Block 1`] = `
<div>
  <!---->
</div>
`;
